samples/javascript_nodejs/14.nlp-with-dispatch/.env

```diff
@@ -1,18 +1,18 @@
 MicrosoftAppId=
 MicrosoftAppPassword=
 
-Luis-Home-Automation-AppId=
-Luis-Home-Automation-Authoringkey=
-Luis-Home-Automation-Region=
+LuisHomeAutomationAppId=
+LuisHomeAutomationAuthoringkey=
+LuisHomeAutomationRegion=
 
-sampleQnaKbId=
-sampleQnaEndpointKey=
-sampleQnaHostname=
+SampleQnaKbId=
```

Comment:
Can we change this to the following? 
QnAKnowledgebaseId,
QnAAuthKey,
QnAEndpointHostName

This mirrors the environment variable names that the QnA Maker team is using in their bot templates.

Reply:
No problem, changed away!

Although with `QnAAuthKey` specifically I'm personally not a huge fan of, because imagine it from a user's perspective, you're told to find the `QnAAuthKey`, there's nothing labeled with that specific label. So the user could think "Is this the key in Azure portal? Or maybe it's something in qnamaker.ai..."
It's specifically the EndpointKey in the settings page at qnamaker.ai and 

![image](https://user-images.githubusercontent.com/35248895/56393988-ca8d2500-61ea-11e9-87cc-0f27adda27da.png)


So imo `QnAEndpointKey` would be better...but I have it as `QnAAuthKey` to mimic the QnA team's bot template :)